src/main/java/com/poiji/bind/Poiji.java

```diff
@@ -370,6 +372,93 @@ private static HSSFPropertyStream deserializerPropertyStream(PoijiExcelType exce
         unmarshaller.unmarshal(type, consumer);
     }
 
+    public static <T> void toExcel(final Sheet sheet,
+                                   final Collection<T> objects,
+                                   final Class<T> type) {
+        toExcel(sheet, objects, type, PoijiOptionsBuilder.settings().build());
+    }
+
+    public static <T> void toExcel(final Sheet sheet,
+                                   final Collection<T> objects,
+                                   final Class<T> type,
+                                   final PoijiOptions options) {
+        Field[] fields = type.getDeclaredFields();
+        List<Field> headers = new ArrayList<>();
+
+        int rowIndex = 0;
+        Row header = sheet.createRow(rowIndex);
```

Comment:
I think, these logics need to move to another class to abstract the core process.

Reply:
Hi @ozlerhakan! Sorry for long delay. I fixed my code by your review.